softwareEngineering/introduction/prosAndCons/q-essay-softwareVsBridges.md

```diff
@@ -5,7 +5,7 @@ Comment on this statement: Building software is cheaper and easier than building
 
 <div slot="answer">
 
-Depends on the size of the software. Manpower required for software is very costly. On the other hand, we can create a very valuable software (e.g. an iPhone application that can make million dollars in a month) with a just a PC and a few days of work!
+Depends on the size of the software. Manpower required for software is very costly. On the other hand, we can create a very valuable software (e.g. an iPhone application that can make million dollars in a month) with just a PC and a few days of work!
```

Comment:
I suggest removing "a" from "a very valuable software" since "software" is uncountable.
Refs: [here](https://english.stackexchange.com/questions/52418/is-a-software-really-never-correct/52458), [here](https://forum.wordreference.com/threads/software-vs-a-software.3216740/)